zombienet_tests/misc/0002-download-polkadot-from-pr.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+set -euxo pipefail
+
+TEMP_DIR=/tmp/zombie
+
+# add /tmp/zombie as first `looking dir` to allow to overrides commands.
+mkdir -p $TEMP_DIR
+export PATH=$TEMP_DIR:$PATH
+
+cd $TEMP_DIR
+#curl -L -O https://gitlab.parity.io/parity/mirrors/polkadot/-/jobs/1810914/artifacts/file/artifacts/polkadot
+curl -L -O $POLKADOT_PR_BIN_URL
```

Comment:
I think we should determine the previous version rather than the latest one, so to ensure we don’t actually test downgrade. This could happen for prs on top of older branches.